### PR TITLE
[Components with more than 1 service] LPS-179756 Refactor ScreenNavigationCategory components from Commerce modules part 6

### PR DESCRIPTION
**Pull request by daiazius**

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:startup

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/constants/CommerceProductAssetScreenNavigationConstants.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/**
```
Comment (rafaprax) [inline modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/constants/CommerceProductAssetScreenNavigationConstants.java:1]: @daiazius, please rename this file to `CPAssetCategoriesScreenNavigationConstants`. Thx

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/constants/CommerceProductScreenNavigationConstants.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+package com.liferay.commerce.product.definitions.web.internal.constants;
+
+/**
+ * @author Pedro Victor Silvestre
+ */
+public class CommerceProductScreenNavigationConstants {
+
+	public static final String CATEGORY_KEY_COMMERCE_PRODUCT_DISPLAY_PAGES =
```
Comment (rafaprax) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/constants/CommerceProductScreenNavigationConstants.java:22]: @daiazius rename this file to `CPDefinitionsScreenNavigationConstants`. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by Kvale1733** [comment]
@daiazius No regressions found.

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by zhangyan0701** [comment]
No regressions found.

**Pull request closed without merging**